pandas/tests/indexing/test_indexing.py

```diff
@@ -623,6 +623,27 @@ def test_index_type_coercion(self):
                     idxr(s2)['0'] = 0
                     assert s2.index.is_object()
 
+    @pytest.mark.parametrize('idx', [
+        Index([1, 1, 3]),
+        Index(['a', 'a', 'b', 'c', 'c', 'd']),
+        Index([1, 1, 1, 2, 2, 3]),
+        Index([1, 1, 2, 3, 1, 2, 3, 3])
+    ])
+    def test_duplicate_int_indexing(self, idx):
+        # GH 17347
+        s = pd.Series(range(len(idx)), idx)
+        dup_idx_filter = s.index.duplicated(keep=False)
```

Comment:
What's the purpose for all this `duplicated` logic? You can just copy the example from the original issue.

Reply:
Hi ! Thank you for your review. This is my first time making a test case contribution, so i just figured it out that i overdid it. I'm fixing it now!